src/WebJobs.Script.Grpc/Extensions/RpcSharedMemoryDataExtensions.cs

```diff
@@ -13,6 +13,11 @@ internal static class RpcSharedMemoryDataExtensions
     {
         internal static async Task<RpcSharedMemory> ToRpcSharedMemoryAsync(this object value, ILogger logger, string invocationId, ISharedMemoryManager sharedMemoryManager)
         {
+            if (value == null)
```

Comment:
This is an optimization but curious on which situation you found this?

Reply:
When you try to access a blob that does not exist, for example.

Reply:
Doing what the corresponding `ToRpc` method [does](https://github.com/Azure/azure-functions-host/blob/1927f5fd736931d2a6407b84d35927f2404eb8dc/src/WebJobs.Script.Grpc/MessageExtensions/GrpcMessageConversionExtensions.cs#L57).